src/main/flight/mixer.h

```diff
@@ -199,6 +200,8 @@ void filterServos(void);
 extern int16_t motor[MAX_SUPPORTED_MOTORS];
 extern int16_t motor_disarmed[MAX_SUPPORTED_MOTORS];
 
+bool motorLimitReached;
```

Comment:
Yes I know, but the whole CF code is like this! I meantioned that many times in the past.